tests/testthat/test-shinytest2-filter_panel.R

```diff
@@ -45,6 +45,10 @@ testthat::test_that("e2e: filtering a module-specific filter is refected in othe
     )
   )
 
+  get_active_filter_vars <- app$get_active_filter_vars()
+  print(app$get_logs())
+  print(app$get_html("#teal-main_ui_container"))
```

Comment:
are you really sure you want to introduce print in the codebase?

Reply:
I should have been clearer on my intention.
My plan was to just temporarily set this up to find more insight on what's going on.
Another alternative is to use the `log_message()` method and write it somewhere so we can retrieve it (need to confirm with IDR where that location would be).

Since then, I also learned that we can run CI test using a branch.
If that's the case, the let's just run it with this branch and close the branch once we learn more.
I'll reach out to IDR to get more info.

Reply:
removed